src/actions/article.js

```diff
@@ -18,7 +29,67 @@ const getAllArticle = () => (dispatch) => {
     payload: articles
   });
 };
+// @loading
+const loading = () => ({
+  type: LOADING
+});
+// @create article
+const createArticle = data => async (dispatch) => {
+  const url = '/api/articles';
+  const arr = data.tag ? data.tag.join(',') : '';
+  const formData = new FormData();
+  formData.append('image', data.image);
+  formData.append('title', data.title);
+  formData.append('body', data.body);
+  formData.append('tag', arr);
+  await dispatch(loading());
+  try {
+    const send = await axios.post(url, formData, config);
+    await dispatch({
+      type: CREATE_ARTICLE,
+      payload: send.data
+    });
+  } catch (error) {
+    await dispatch({
+      type: ARTICLE_ERROR,
```

Comment:
Rename this type to CREATE_ARTICLE_FAILURE

Reply:
thank you @zellymiriam